### ISSUE TITLE
v1.100 - Kerning only works on Default (DFLT)

### ISSUE THREAD

**Issue by kenmcd** [DCWelch/Funiko#1]
First, love the font. Lots of personality.

But, the kerning is only working for Default (DFLT).
Opened the v1.100 release in Windows Font Viewer...
and the `r` spacing seemed off.

![Windows-Font-Viewer](https://github.com/user-attachments/assets/b5b796f6-6431-4e2c-8b42-25ab70e016d3)

So a little investigation showed the kerning was not working.

![2024-10-02_14-12-56](https://github.com/user-attachments/assets/48f5a201-4c44-49f8-bb2a-bd6597ebc6c2)

Very nice font.
Look forward to the final!

**Comment by DCWelch** [comment]
Thank you for the feedback @kenmcd! I am not an expert in the area, so I may need some help...

I think you are using FontGoggles to view the font and see the difference between the "dflt" vs. "latn" preview? I don't have a Mac, so I am not sure if I can replicate the exact same setup you have. When I tried to replicate your findings using FontForge, I did not see the option to separate the default and Latin scripts when previewing the font:

![image](https://github.com/user-attachments/assets/1311be9c-07d1-4b77-8085-4debb43ab732)

I can see how the kerning doesn't appear to be working for the Latin script in your example, but I don't understand why the two scripts are able to be viewed independently in the first place? In my head, the Latin script is the default script, so I am a bit confused. I'll see if I can get to the bottom of it in the meantime.

Let me know if you have any guidance!

**Comment by DCWelch** [comment]
Okay, I believe I have figured out how to explicitly enforce the kerning for both the DFLT and latn scripts. I still do not really understand why they are different, but I can now view each independently in FontForge's previewer and it appears to me that the same kerning applies to both now.

@kenmcd - I have attached the new TTF file (built locally following the recent change https://github.com/DCWelch/Funiko/commit/aef548ba945214508e519d2d069824b889157f88). Do you have time to verify if this has resolved things? Thanks in advance!

[FunikoRoman-Regular.zip](https://github.com/user-attachments/files/17237785/FunikoRoman-Regular.zip)

**Comment by kenmcd** [comment]
> @kenmcd - I have attached the new TTF file (built locally following the recent change [aef548b](https://github.com/DCWelch/Funiko/commit/aef548ba945214508e519d2d069824b889157f88)). Do you have time to verify if this has resolved things? Thanks in advance!
> 
> [FunikoRoman-Regular.zip](https://github.com/user-attachments/files/17237785/FunikoRoman-Regular.zip)

That works better.
The kerning works correctly now for both DFLT and latn.

My animated test image above is from testing the font in FontCreator on Windows 10.
I am not really familiar with FontForge so I am not sure of good settings for this issue.

GF does have a desired format for the languagesystem settings.
I have seen it discussed previously.
I will go find it. 
Getting it right for GF may involve some settings for their build process.
May be in the FontBakery report.
May be OK as is now.
I will go check it out and see if I can help here.

**Comment by DCWelch** [comment]
I went and updated release v1.100 and the PR assuming this is a viable solution. Let me know if there is anything else you think needs to be done, but I think the latn script kerning issue has been addressed. I will close this issue for now, but am happy to open it back up if I have misunderstood something.

If you find anything else that needs to change in the format of the languagesystem settings or something, I think one of us can open up another issue  :)

Thank you for your help!
